pkg/cloudprovider/suite_test.go

```diff
@@ -314,6 +327,23 @@ var _ = Describe("CloudProvider", func() {
 			Expect(err).ToNot(HaveOccurred())
 			Expect(isDrifted).To(BeTrue())
 		})
+		It("should return drifted if the securitygroup is not valid", func() {
```

Comment:
We should ensure the opposite of this. That is, that we don't return drifted when the security groups are equal